lib/rsvp/promise.js

```diff
@@ -114,7 +114,7 @@ function needsNew() {
   Unlike callbacks, promises are great composable primitives.
 
   ```js
-  Promise.all([
+  RSVP.all([
```

Comment:
Is the `RSVP.Promise.all` syntax preferable to simply `RSVP.all`? The documentation claims that it's a "convenient alias":

http://emberjs.com/api/classes/RSVP.html#method_all